### PR TITLE
Add migration guide for Stack AzureRm 1.2.11

### PR DESCRIPTION
**Pull request by viananth**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [ ] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

File: documentation/release-notes/Stack/migration-guide.1.2.11.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Table of Contents
+1. [Removal of Force parameters](#removal-of-force-parameters)
```
Comment (markcowl) [inline documentation/release-notes/Stack/migration-guide.1.2.11.md]: I think you will want to add in the changes from the migration guides for 2.X and 3.X powershell for the modules you are using (Profile esp.): https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.2.0.0.md, https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.3.0.0.md, https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.4.0.0.md

File: documentation/release-notes/Stack/migration-guide.1.2.11.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+2. [Change of Tag parameters](#change-of-tag-parameters)
+3. [Breaking Changes to Profile Cmdlets](#breaking-changes-to-profile-cmdlets)
```
Comment (bganapa) [inline documentation/release-notes/Stack/migration-guide.1.2.11.md]: We also have changes in the Get-AzureRmSubscription and Select-AzureRmSubscription cmdlets differring in outputs from 1.2.10

We cannot pipe azurestack subscription model to Select-AzureRmSubscription, we need to call these out

**Comment by bganapa** [comment]
@markcowl  vishnu added the Profile Changes from 3.0 and 4.0

**Comment by bganapa** [comment]
Thanks again!

**Pull request merged**